src/config/index.js

```diff
@@ -0,0 +1,11 @@
+module.exports = Object.freeze({
```

Comment:
What is Object.freeze() it's the first time i see that.

Reply:
Basically this will freeze the object that you have, you cannot add something else while running, you cannot remove properties, nor modify any of them.